src/Tribe/Main.php

```diff
@@ -2583,6 +2582,8 @@ public function redirect_past_upcoming_view_urls() {
 		 * @return string The link.
 		 */
 		public function getLink( $type = 'home', $secondary = false, $term = null, $featured = null ) {
+			static $cache_event_url_slugs = [];
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: ac53cae7b6a1cb5b69ee2280b5f6c909</sub>